packages/popup/src/components/DappFooter.js

```diff
@@ -56,6 +58,10 @@ function DappFooter({
       case WALLET_SWITCH_CONFLUX_CHAIN:
         params.chainConfig = req.params
         break
+      case CFX_SEND_TRANSACTION:
+      case ETH_SEND_TRANSACTION:
+        params.tx = req.params
```

Comment:
so the `req` here is from `usePendingAuthReq`
but user can change the tx data with the send tx UI
is this `req` the user-altered one?

Reply:
user can change tx data with the send UI in wallet but I forget the dapp send tx, I will add a task in Clickup to do the dapp send tx data changing.

Reply:
https://app.clickup.com/t/3719212/PORTAL-1441

Reply:
done